optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/score/director/TestdataComparableDifferentValuesCalculator.java

```diff
@@ -0,0 +1,34 @@
+package org.optaplanner.core.impl.testdata.score.director;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.optaplanner.core.api.score.buildin.simple.SimpleScore;
+import org.optaplanner.core.impl.score.director.easy.EasyScoreCalculator;
+import org.optaplanner.core.impl.testdata.domain.TestdataValue;
+import org.optaplanner.core.impl.testdata.domain.comparable.TestdataEntityWithDifficultyComparator;
+import org.optaplanner.core.impl.testdata.domain.comparable.TestdataSolutionWithDifficultyComparatorEntity;
+
+public class TestdataComparableDifferentValuesCalculator implements EasyScoreCalculator<TestdataSolutionWithDifficultyComparatorEntity> {
```

Comment:
Does not belong in package "testdata", that package is reserved for domain objects.

Put it near the test, either as a static class inside the test, or next to it.